.drone.star

```diff
@@ -1,24 +1,47 @@
+repo = "spritsail/alpine"
+archs = ["amd64", "arm64"]
+versions = ["edge", "3.14", "3.15", "3.16"]
+branches = ["master"]
+
 def main(ctx):
-  return [
-    step("3.15"),
-    step("3.16",["latest"]),
-    step("edge"),
-  ]
+  builds = []
+
+  for ver in versions:
+    depends_on = []
+    srctpl = "drone/%s/${DRONE_BUILD_NUMBER}:%s-ARCH" % (ctx.repo.slug, ver)
+    for arch in archs:
+      key = "build-%s-%s" % (ver, arch)
+      repo = "drone/%s/${DRONE_BUILD_NUMBER}:%s-%s" % (ctx.repo.slug, ver, arch)
+      builds.append(step(ver, arch, key, repo))
+      depends_on.append(key)
+
+    if ctx.build.branch in branches:
+      tags = []
+      if ver == versions[-1]:
+        tags.append("latest")
+      builds.append(publish(ver, srctpl, depends_on, tags))
 
-def step(alpinever,tags=[]):
-  vertest = "grep -q '%s' /etc/alpine-release && " % alpinever if alpinever != "edge" else ""
+  return builds
+
+def step(ver, arch, key, repo):
+  vertest = "grep -q '%s' /etc/alpine-release && " % ver if ver != "edge" else ""
```

Comment:
Do we even need this any more?

Reply:
Let's find out

Reply:
```
root@1b79854e629e /> grep -q edge /etc/alpine-release
root@1b79854e629e /> echo $?
1
root@1b79854e629e /> cat /etc/alpine-release
3.16.0_alpha20220328
```

Reply:
ah, so edge doesn't have a release file. That tracks.